pkg/cmd/roachtest/jepsen.go

```diff
@@ -270,7 +277,7 @@ func registerJepsen(r *registry) {
 
 	for i := range groups {
 		spec := testSpec{
-			Name:  fmt.Sprintf("jepsen/%d", i+1),
+			Name:  fmt.Sprintf("jepsen-group-%d", i+1),
```

Comment:
Maybe just change the separator from `/` to `-`? These test names are already verbose so adding `-group` might be annoying (and IIRC there's a length limit somewhere but I'm not sure what it is)